src/node.cc

```diff
@@ -1383,27 +1384,28 @@ void AppendExceptionLine(Environment* env,
                      linenum,
                      sourceline_string);
   CHECK_GE(off, 0);
```

Comment:
I'm testing on Windows so there must be a `#define snprintf _snprintf` somewhere because the documentation on MSDN for `_snprintf` says:

> Let len be the length of the formatted data string, not including the terminating null. len and count are in bytes for _snprintf, wide characters for _snwprintf.
> ....
> If len > count, count characters are stored in buffer, no null-terminator is appended, and a negative value is returned.